apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/FissClaimTransformerV2.java

```diff
@@ -165,8 +165,8 @@ private static List<Claim.SupportingInformationComponent> getSupportingInfo(
 
   private static Period getBillablePeriod(PreAdjFissClaim claimGroup) {
     return new Period()
-        .setStart(localDateToDate(claimGroup.getStmtCovToDate()))
-        .setEnd(localDateToDate(claimGroup.getStmtCovFromDate()));
+        .setStart(localDateToDate(claimGroup.getStmtCovFromDate()))
+        .setEnd(localDateToDate(claimGroup.getStmtCovToDate()));
```

Comment:
Please write a test covering this (since the lack of one likely led to this bug)